reference/test/solver/ir_kernels.cpp

```diff
@@ -209,4 +209,104 @@ TYPED_TEST(Ir, SolvesConjTransposedTriangularSystem)
 }
 
 
+TYPED_TEST(Ir, RichardsonSolvesTriangularSystem)
+{
+    using Mtx = typename TestFixture::Mtx;
+    using value_type = typename TestFixture::value_type;
+    auto solver = gko::solver::Ir<value_type>::build()
+                      .with_criteria(
+                          gko::stop::Iteration::build().with_max_iters(100u).on(
+                              this->exec),
+                          gko::stop::ResidualNormReduction<value_type>::build()
+                              .with_reduction_factor(r<value_type>::value)
+                              .on(this->exec))
+                      .with_relaxation(value_type{1})
```

Comment:
Does the test also converge with another relaxation factor?

Reply:
Yes, it should be 0.9. I forget to change it.